make/test-integration.mk

```diff
@@ -1,4 +1,6 @@
-COVER_PACKAGES = $(shell go list ./... | grep -v mock)
+space = $(subst ,, )
+comma = ,
+COVER_PACKAGES = $(subst $(space),$(comma),$(shell go list ./... | grep -v mock))
```

Comment:
I'd suggest doing it in shell instead of the `make` hack:

```
$(shell go list ./... | grep -v mock | tr ' ' ',')
```

I think that we should exclude proto-generated things if we can, but we can iterate on this later.

Reply:
Done